.tekton/instaslice-daemonset-pull-request.yaml

```diff
@@ -7,8 +7,10 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "main" && files.all.exists(path, !path.matches('bundle*|tests/*|.tekton/*bundle*'))
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" &&
+      target_branch == "main" &&
+      files.all.exists(path, path.matches('internal/controller/daemonset/.*|cmd/daemonset/.*|.tekton/.*daemonset.*|Dockerfile.daemonset-ocp'))
```

Comment:
Do you we have a non ocp docker file for daemonset that should also be included here?

Reply:
I intentionally left the upstream files out, but that probably doesn't make sense since the CI will be for all upstream PRs as well.  I will change that.